demo/index.html

```diff
@@ -18,24 +18,24 @@
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <script>
       (function() {
-        window.ensureMediaDevices = new Promise(function(resolve) {
-          if (navigator.mediaDevices == null) {
-            // try to load the polyfill from the NPM directory first
-            var npmScript = document.createElement('script');
-            npmScript.src = '../../webrtc-adapter/out/adapter.js';
-            npmScript.onload = resolve;
-            npmScript.onerror = function() {
-              // on error try loading it from the bower directory
-              var bowerScript = document.createElement('script');
-              bowerScript.src = '../../webrtc-adapter/release/adapter.js';
-              bowerScript.onload = resolve;
-              document.body.appendChild(bowerScript);
-            }
+      window.ensureMediaDevices = new Promise(function(resolve) {
+        if (navigator.mediaDevices == null) {
+          // try to load the polyfill from the NPM directory first
+          var npmScript = document.createElement('script');
+          npmScript.src = '../../webrtc-adapter/out/adapter.js';
+          npmScript.onload = resolve;
+          npmScript.onerror = function() {
+            // on error try loading it from the bower directory
+            var bowerScript = document.createElement('script');
+            bowerScript.src = '../../webrtc-adapter/release/adapter.js';
+            bowerScript.onload = resolve;
+            document.body.appendChild(bowerScript);
             document.body.appendChild(npmScript);
-          } else {
-            resolve();
           }
```

Comment:
This looks like it shouldn't have moved.

Reply:
Realized my comment doesn't make much sense without the `?w=1` view... I think the `document.body.appendChild(npmScript);` needs to be outside the `onerror` handler for it to try to load.